tests/test_library.py

```diff
@@ -300,6 +300,34 @@ def test_library_MovieSection_collection_exception(movies):
         movies.collection("Does Not Exists")
 
 
+@pytest.mark.authenticated
+def test_library_MovieSection_managedHubs(movies):
+    recommendations = movies.managedHubs()
+    with pytest.raises(BadRequest):
+        recommendations[0].remove()
+    first = recommendations[0]
+    first.promoteRecommended().promoteHome().prmoteShared().reload()
+    assert first.promotedToRecommended is True
+    assert first.promotedToOwnHome is True
+    assert first.promotedToSharedHome is True
+    first.demoteRecommended().demoteHome().demoteShared().reload()
+    assert first.promotedToRecommended is False
+    assert first.promotedToOwnHome is False
+    assert first.promotedToSharedHome is False
+    last = recommendations[-1]
+    last.move()
+    recommendations = movies.managedHubs()
+    assert first.identifier == recommendations[1].identifier
+    assert last.identifier == recommendations[0].identifier
+    last.move(after=first)
+    recommendations = movies.managedHubs()
+    assert first.identifier == recommendations[0].identifier
+    assert last.identifier == recommendations[1].identifier
+    movies.resetManagedHubs()
```

Comment:
Need to get the recommendations again.
```suggestion
    movies.resetManagedHubs()
    recommendations = movies.managedHubs()
```